tgui-next/packages/tgui/interfaces/Cargo.js

```diff
@@ -0,0 +1,338 @@
+                  : pack.cost) + ' credits'}
+                tooltip={pack.desc}
+                tooltipPosition="left"
+                onClick={() => act(ref, 'add', {
+                  id: pack.id,
+                })} />
+            </td>
+          </tr>
+        ))}
+      </table>
+    );
+  };
+  return (
+    <Tabs vertical>
+      {map(supply => {
+        const name = supply.name;
+        return (
+          <Tabs.Tab key={name} label={name}>
+            {renderTab}
+          </Tabs.Tab>
+        );
+      })(supplies)}
+    </Tabs>
+  );
+};
+
+const Requests = props => {
+  const { state, requests } = props;
+  const { config, data } = state;
+  const { ref } = config;
+  if (requests.length === 0) {
+    return (
+      <Box color="good">
+        No Requests
+      </Box>
+    );
+  }
+  // Labeled list reimplementation to squeeze extra columns out of it
+  return (
+    <table className="LabeledList">
+      {requests.map(request => (
+        <Fragment key={request.id}>
+          <tr className="LabeledList__row candystripe">
+            <td className="LabeledList__cell LabeledList__label">
+              #{request.id}:
+            </td>
+            <td className="LabeledList__cell LabeledList__content">
+              {request.object}
+            </td>
+            <td className="LabeledList__cell">
+              By <b>{request.orderer}</b>
+            </td>
+            <td className="LabeledList__cell">
+              <i>{request.reason}</i>
+            </td>
+            <td className="LabeledList__cell LabeledList__buttons">
+              {request.cost} credits
+              {' '}
+              {!data.requestonly && (
+                <Fragment>
+                  <Button
+                    icon="check"
+                    color="good"
+                    onClick={() => act(ref, 'approve', {
+                      id: request.id,
+                    })} />
+                  <Button
+                    icon="times"
+                    color="bad"
+                    onClick={() => act(ref, 'deny', {
+                      id: request.id,
+                    })} />
+                </Fragment>
+              )}
+            </td>
+          </tr>
+        </Fragment>
+      ))}
+    </table>
+  );
+};
+
+const Cart = props => {
+  const { state, cart } = props;
+  const { config, data } = state;
+  const { ref } = config;
+  return (
+    <Fragment>
+      {cart.length === 0 && 'Nothing in cart'}
+      {cart.length > 0 && (
+        <LabeledList>
+          {cart.map(entry => (
+            <LabeledList.Item
+              key={entry.id}
+              className="candystripe"
+              label={'#' + entry.id}
+              buttons={(
+                <Fragment>
+                  <Box inline mx={2}>
+                    {!!entry.paid && (<b>[Paid Privately]</b>)}
+                    {' '}
+                    {entry.cost} credits
+                  </Box>
```

Comment:
you should remove this for now and recompile tgui-next, we don't have currency on this server

Reply:
Been trying to do that, but having some problems with indendation